src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php

```diff
@@ -75,7 +77,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
             array('OPcache', extension_loaded('Zend OPcache') && ini_get('opcache.enable') ? 'true' : 'false'),
             array('APCu', extension_loaded('apcu') && ini_get('apc.enabled') ? 'true' : 'false'),
             array('Xdebug', extension_loaded('xdebug') ? 'true' : 'false'),
-        ));
+        );
+
+        if ($dotenv = self::getDotEnvVars()) {
+            $rows = array_merge($rows, array(
+                new TableSeparator(),
+                array('<info>Dotenv</>'),
```

Comment:
`Dotenv` doesn't mean much unless you know what is Dotenv. Could we rename this to `Environment variables loaded by Symfony` or something like that? Thanks.

Reply:
Fair point. Let me think of better name, yet i could live with "Dotenv" :) more suggestions welcome.

Reply:
What about `Environment variables (dotenv)`?

It's enabled by using `DotEnv` directly, so should be clear.

Reply:
I'd use `Environment variables (.env)` which seems more "correct". The user defined the vars in a `.env` file and they don't care about how they are loaded.

Reply:
`Managed environment variables`